requirements/edx/base.txt

```diff
@@ -6,6 +6,7 @@
 
 beautifulsoup4==4.1.3
 beautifulsoup==3.2.1
+before_after==0.1.3
```

Comment:
@efischer19 can this go in requirements/edx/development.txt instead of base.txt?